packages/core/src/App/Containers/NotificationsDialog/notifications-dialog.jsx

```diff
@@ -151,5 +151,5 @@ NotificationsDialog.propTypes = {
 };
 
 export default connect(({ ui }) => ({
-    notifications: ui.notifications,
+    notifications: ui.notifications.filter(message => message.type !== 'news'),
```

Comment:
since this is used in multiple places can we add a `computed` prop in ui store instead?

Reply:
Sure, will do.